pkg/ccl/changefeedccl/helpers_test.go

```diff
@@ -754,6 +763,68 @@ func withKnobsFn(fn updateKnobsFn) feedTestOption {
 	return func(opts *feedTestOptions) { opts.knobsFn = fn }
 }
 
+// withFailOnChangefeedErr is a feedTestOption that will cause the test to fail
+// if a changefeed error is encountered.
+func withAllowChangefeedErr(
+	reason string, /*for documentation only*/
```

Comment:
How much extra work would it be to accept a list of allowed errors and still fail if it's not one of those (or the default list of allowed errors)?

Reply:
It's been a while since I left my original comment so I don't really remember what I was concerned about. Maybe that this was too permissive and we might still end up with timeouts? Looking at the list again today, it seems fine to me.